resources/automatic_software_updates.rb

```diff
@@ -0,0 +1,58 @@
+resource_name :automatic_software_updates
+
+property :check, [TrueClass, FalseClass]
+
```

Comment:
@v-anshie Nit: It'd be nice if the newlines here were removed to match the formatting of the other resources.

Reply:
Good catch I usually like to match and just didn't notice. Pushed the change.